PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/mysql/MySQLPropertyView.java

```diff
@@ -0,0 +1,254 @@
+import com.microsoft.azure.management.mysql.v2020_01_01.ServerState;
+import com.microsoft.azure.management.mysql.v2020_01_01.implementation.DatabaseInner;
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.intellij.common.AzureHideableTitledSeparator;
+import com.microsoft.azure.toolkit.intellij.mysql.ConnectionSecurityPanel;
+import com.microsoft.azure.toolkit.intellij.mysql.ConnectionStringsOutput;
+import com.microsoft.azure.toolkit.intellij.mysql.DatabaseComboBox;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azuretools.azurecommons.util.Utils;
+import com.microsoft.azuretools.core.mvp.model.AzureMvpModel;
+import com.microsoft.azuretools.core.mvp.model.mysql.MySQLMvpModel;
+import com.microsoft.intellij.helpers.base.BaseEditor;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLProperty;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLPropertyMvpView;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLPropertyViewPresenter;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ItemEvent;
+
+public class MySQLPropertyView extends BaseEditor implements MySQLPropertyMvpView {
+
+    public static final String ID = "com.microsoft.intellij.helpers.mysql.AzureMySQLPropertyView";
+    private final MySQLPropertyViewPresenter<MySQLPropertyView> propertyViewPresenter;
+
+    private AzureHideableTitledSeparator overviewSeparator;
+    private MySQLPropertryOverview overview;
+    private AzureHideableTitledSeparator connectionSecuritySeparator;
+    private ConnectionSecurityPanel connectionSecurity;
+    private AzureHideableTitledSeparator connectionStringsSeparator;
+    private ConnectionStringsOutput connectionStringsJDBC;
+    private ConnectionStringsOutput connectionStringsSpring;
+    private JPanel rootPanel;
+    private JPanel contextPanel;
+    private JScrollPane scrollPane;
+    private MySQLPropertyAction propertyActionPanel;
+    private DatabaseComboBox databaseComboBox;
+    private JLabel databaseLabel;
+    public static final String MYSQL_OUTPUT_TEXT_PATTERN_SPRING =
+            "spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver" + StringUtils.LF +
+                    "spring.datasource.url=jdbc:mysql://localhost:3306/%s?useSSL=true&requireSSL=false" + StringUtils.LF +
+                    "spring.datasource.username=qianjin@qianjin-mysql-01" + StringUtils.LF + "spring.datasource.password={your_password}";
+
+    public static final String MYSQL_OUTPUT_TEXT_PATTERN_JDBC =
+            "String url =\"jdbc:mysql://qianjin-mysql-10.mysql.database.azure.com:3306/%s?useSSL=true&requireSSL=false\";" + StringUtils.LF +
```

Comment:
use placeholder instead of your own name

Reply:
updated.